__tests__/app.test.js

```diff
@@ -85,4 +86,28 @@ describe( "/api/articles/:article_id", ()=> {
           });
     
 })
-    
\ No newline at end of file
+    
+
+describe( "/api/articles", ()=> {
+    it("GET 200 sends all articles from test data in an array in descending order with their properties (apart from body)", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articles).toBeSortedBy("created_at", {descending: true});
+            expect(body.articles).toHaveLength(13);
+            body.articles.forEach((article)=> {
+            expect(typeof article.author).toBe("string")
+            expect(typeof article.title).toBe("string")
+            expect(typeof article.article_id).toBe("number")
+            expect(typeof article.topic).toBe("string")
+            expect(typeof article.created_at).toBe("string")
+            expect(typeof article.votes).toBe("number")
+            expect(typeof article.article_img_url).toBe("string")
+            expect(typeof Number(article.comment_count)).toBe("number")
+            expect(Object.keys(article).includes('body')).toBe(false)
```

Comment:
i wouldnt change the data type of your response then check it - if you want it to be a number, you need to make sure you've sent it back as a number